### PR TITLE
Add example for Fee Preferences and proxy extrinsic which is a fpass proxy call

### PR DESCRIPTION
**Pull request by KarishmaBothara**
## Summary

Closes https://centralitydev.atlassian.net/browse/TRN-223

## Checklist

- [x] Add description
- [x] Tag related issue(s)

### CODE REVIEW

File: examples/substrate/use-feeProxy/src/feeProxyWithProxyExtrinsic.ts
Comment (ken-centrality) [inline examples/substrate/use-feeProxy/src/feeProxyWithProxyExtrinsic.ts:1]: example looks good but needs to be trimmed down a bit, no need for the FPass creation as well as minting assets

File: examples/substrate/use-feeProxy/src/feeProxyWithProxyExtrinsic.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  // can be any extrinsic, using `system.remarkWithEvent` for simplicity
+  const innerCall = api.tx.system.remarkWithEvent("Hello World");
+
+  const proxyExtrinsic = api.tx.futurepass.proxyExtrinsic(
+    fpassAddress,
+    innerCall
+  );
+  const maxPayment = 1000000;
+  const feeproxiedCall = api.tx.feeProxy.callWithFeePreferences(
```
Comment (ken-centrality) [inline examples/substrate/use-feeProxy/src/feeProxyWithProxyExtrinsic.ts:35]: nit:

```suggestion
  const feeProxiedCall = api.tx.feeProxy.callWithFeePreferences(
```

**Review by ken-centrality — APPROVED** [review]

File: examples/substrate/use-feeProxy/src/feeProxyWithProxyExtrinsic.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+	const innerCall = api.tx.system.remarkWithEvent("Hello World");
+
+	const proxyExtrinsic = api.tx.futurepass.proxyExtrinsic(fpassAddress, innerCall);
+	const maxPayment = 1000000;
+	const feeProxiedCall = api.tx.feeProxy.callWithFeePreferences(
+		paymentAsset,
+		maxPayment,
+		proxyExtrinsic
+	);
+	const { result } = await sendExtrinsic(feeproxiedCall, caller, {
```
Comment (vldolot) [inline examples/substrate/use-feeProxy/src/feeProxyWithProxyExtrinsic.ts:35]: This `feeproxiedCall` needs to be changed as well to `feeProxiedCall`

Comment (KarishmaBothara) [reply examples/substrate/use-feeProxy/src/feeProxyWithProxyExtrinsic.ts]: thanks.. updated..

**Review by vldolot — APPROVED** [review]

**Pull request merged**